app/models/web_document.rb

```diff
@@ -18,6 +18,14 @@ def language
     @language ||= extract_language&.downcase
   end
 
+  def created
+    timestamp(extract_created)
+  end
+
+  def changed
```

Comment:
I'm using "changed" here to mirror the I14y API, which takes `created` and `changed`.